komga-webui/src/components/dialogs/EditSeriesDialog.vue

```diff
@@ -311,6 +315,32 @@
               </v-container>
             </v-card>
           </v-tab-item>
+
+          <!--  Tab: Thumbnails  -->
+          <v-tab-item v-if="single">
+            <v-card flat>
+              <v-container fluid>
+                <!-- Upload -->
+                <v-row>
+                  <drop-zone @onInputChange="inputChangeHandler" />
+                </v-row>
+
+                <!-- Gallery -->
+                <v-container class="d-flex flex-row flex-wrap">
+                  <thumbnail-card
+                    v-for="item in [...uploadQueue, ...seriesThumbnails]"
+                    :key="getKey(item)"
+                    :item="item"
+                    :selected="isSelected(item)"
+                    :toBeDeleted="toBeDeleted(item)"
+                    @onSelectThumbnail="selectThumbnailHandler"
+                    @onDeleteThumbnail="deleteThumbnailHandler"
```

Comment:
Should use kebab-case for events, see https://vuejs.org/v2/guide/components-custom-events.html#Event-Names